libexec/cli/stop.help

```diff
@@ -0,0 +1,21 @@
+USAGE: singularity [...] stop [list options...] <container path> <instance name>
+
+Stop a named instance of the given container image.
+
+LIST OPTIONS:
+    -a/--all               If running as root, stop the specified instance
```

Comment:
Is this working?  I can't seem to get either `--all` or `stop-all` to work as advertised when run as root.  Also maybe a very quick word of caution here so the admin knows that `--all` may have the unexpected side effect of creating nasty emails in one's inbox (depending on the system).